content/index.yaml

```diff
@@ -21,4 +21,10 @@ usecases:
     id: 3
     category: Miscellaneous
     name: Infrastructure in support of EUropean Climate Prediction
-    description: P. Kalverla et al. Netherlands eScience Center.
\ No newline at end of file
+    description: P. Kalverla et al. Netherlands eScience Center.
+
+  coastal:
+    id: 4
+    category: Application of EUCP innovations (WP4)
+    name: Sandy beach erosion induced by sea level rise
+    description: P. Athanasiou, et al.Deltares.
```

Comment:
```suggestion
    description: P. Athanasiou, et al. Deltares.
```

Reply:
Thanks Peter, for finding the typo. It is corrected now!